Automattic/jetpack
# SSO: Fix Button Text Alignment on Mobile

Fixes #18658

#### Changes proposed in this Pull Request:

* Fixes issue with login alignment on mobile.

#### Jetpack product discussion
N/a



#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Create a Jetpack site and enable SSO/WordPress.com Login
* Log out (or open the site in an incognito window) and visit WP-Admin
* Resize the screen to less than 782px
* See how the text in the login button isn't vertically aligned 
* Apply this branch
* Refresh and see how the button is now centered.

Checked multiple browsers and it works fine. Did some digging, but unsure why the line height is not-needed now or why it was added. Maybe a CSS wizard can shed some light ¯\_(ツ)_/¯

**Before**


![Screen Shot on 2021-02-09 at 17:23:44](https://user-images.githubusercontent.com/33553323/107436912-30ebe180-6afc-11eb-8eb8-ad12a3a2c3a1.png)

**After**

![Screen Shot on 2021-02-09 at 17:28:23](https://user-images.githubusercontent.com/33553323/107436929-39441c80-6afc-11eb-8100-3e5b3fd8b423.png)

#### Proposed changelog entry for your changes:


* fixes login button alignment when SSO is enabled
